JeMPI_Apps/JeMPI_UI/src/components/recordDetails/RecordDetails.tsx

```diff
@@ -30,7 +30,7 @@ import { PatientRecord, GoldenRecord, AnyRecord } from 'types/PatientRecord'
 import { sortColumns } from 'utils/helpers'
 import getCellComponent from 'components/shared/getCellComponent'
 import { AUDIT_TRAIL_COLUMNS } from 'utils/constants'
-import { AuditTrail } from 'types/AuditTrail'
+import { AuditTrail, ExpandedAuditTrail } from 'types/AuditTrail'
```

Comment:
I dont think you should create a new type. The change is too small (i.e not worth the extra complexities) . Just update the current version of the `AuditTrail` type with the additional type.